bson/src/main/org/bson/codecs/configuration/mapper/ClassModelCodec.java

```diff
@@ -59,12 +59,10 @@ public T decode(final BsonReader reader, final DecoderContext decoderContext) {
     }
 
     @Override
-    public void encode(final BsonWriter writer, final Object value, final EncoderContext encoderContext) {
+    public void encode(final BsonWriter writer, final Object entity, final EncoderContext encoderContext) {
```

Comment:
I liked value. entity implies something in DDD, and an entity is not a value type. "value" seems sufficiently generic.